pkg/minikube/cluster/cluster_test.go

```diff
@@ -50,3 +53,52 @@ func TestLocalBoot2DockerURL(t *testing.T) {
 		t.Fatalf("Expected URL : %s", localISOUrl)
 	}
 }
+
+func TestHostGetLogs(t *testing.T) {
+	api := tests.NewMockAPI()
```

Comment:
Can you run me through this test and explain to me how this is a test for the tailing functionality? Or what you are testing at all here?

At best you are testing that the log command w/ and w/o the -f option is received by the ssh server. But what does that prove really? Am I missing something here?